tensorflow_hub/tools/make_image_classifier/make_image_classifier_lib.py

```diff
@@ -38,7 +38,7 @@ def get_default_image_dir():
 class HParams(
     collections.namedtuple("HParams", [
         "train_epochs", "do_fine_tuning", "batch_size", "learning_rate",
-        "momentum", "dropout_rate"
+        "momentum", "dropout_rate", "set_memory_growth"
```

Comment:
Please do not add the new flag here. Keep it completely in the main script, and set the obvious default (`False`) there.

Background: The `HParams` exposed by the `lib` control how the model is built and trained by the code in the `lib`. The GPU configuration does not happen in the `lib` but at the start of the main program (which is reasonable, because it really is global state of the execution environment; although reuse by `lib` users might come up as a later request).